apps/prairielearn/src/lib/question.js

```diff
@@ -1164,9 +1199,8 @@ function testQuestion(
  * @param {Object} course - The course for the variant.
  * @param {string} test_type - The type of test to run.  Should be one of 'correct', 'incorrect', or 'invalid'.
  * @param {string} authn_user_id - The currently authenticated user.
- * @param {function} callback - A callback(err, success) function.
  */
-function runTest(
+async function runTest(
```

Comment:
I opted to refactor this to an async function; better reviewed with whitespace changes hidden.